recipes/eigenpy/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
```

Comment:
You might should consider `ninja` instead of `make`. This will make better use of the available cores. You can also use `make -j${CPUCOUNT}` for parallelization but `ninja` generally is a bit faster.

Reply:
Note: You can do that later on the feedstock, not required for a merge here.

Reply:
OK thanks I'll consider that.